src/imap.rs

```diff
@@ -2076,6 +2081,14 @@ pub async fn get_config_last_seen_uid(context: &Context, folder: &str) -> Result
     }
 }
 
+async fn should_ignore_folder(context: &Context, folder: &str) -> Result<bool> {
+    Ok(context.get_config_bool(Config::OnlyFetchMvbox).await?
+            && context.is_mvbox(folder).await?
+            // Even if OnlyFetchMvbox is set, we have a look at the spam folder
+            // in case an answer to a known message was put into spam:
+            && context.is_spam_folder(folder).await?)
```

Comment:
leaving all other discussion to the side, is this condition correct?  It seems to me it would never be `true` unless you managed to configure your mvbox to be spam folder.

Reply:
It should be (onlyfetchmvbox && !is_mvbox) || is_spam_folder

Reply:
Oops, right